.github/workflows/pre-commit.yml

```
@@ -74,4 +74,5 @@ jobs:
 
     - env:
         CHIA_MANAGE_CLVM_CHECK_USE_CACHE: "false"
+        CHIA_MANAGE_MYPY_CHECK_EXCLUSIONS: true
```

I think it kind of works regardless, but if you intend the env var to be the string `"true"` then I think quoting would be the explicit way to do that.

```suggestion
        CHIA_MANAGE_MYPY_CHECK_EXCLUSIONS: "true"
```

---

I added it for consistency with the one above, but why is that better if it works without?